aries_cloudagent/commands/upgrade.py

```diff
@@ -136,68 +168,64 @@ async def upgrade(settings: dict):
                         f"{upgrade_from_version} as --from-version from "
                         "the config."
                     )
-        if upgrade_from_version == upgrade_to_version:
+        upgrade_version_in_config = get_upgrade_version_list(
+            sorted_versions_found_in_config, upgrade_from_version
+        )
+        force_upgrade_flag = root_profile.settings.get("upgrade.force_upgrade") or False
```

Comment:
The `force` upgrade should apply when the “—from-version” command line arg is before the `from` that is pulled from storage.  When the `force` is in effect, the command line arg overrides the secure storage, otherwise the opposite is used — the storage `from` overrides the command line `from`.